Makefile

```
@@ -2,10 +2,22 @@
+BUILD_LDFLAGS ?= "-X github.com/whywaita/rfid-poker/pkg/version.gitVersion=$(GIT_VERSION) -X github.com/whywaita/rfid-poker/pkg/version.gitCommit=$(CURRENT_REVISION)"
+BINARY_NAME = "cmd"
 
 DOCKER_IMAGE_NAME = "rfid-poker"
 
+bin:
+	mkdir -p $@
+
+.PHOHY: bin/rdid-poker
+bin/rdid-poker: bin
+	go build -ldflags $(BUILD_LDFLAGS) -o $@ cmd/server/main.go
+.PHONY: bin/wired-client
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)